spec/controllers/authentication_controller_spec.rb

```diff
@@ -13,11 +13,29 @@
     it 'creates a new person' do
       expect { subject }.to change { Person.count }.by(1)
     end
-    it 'redirects the person to the root url' do
-      expect(subject).to redirect_to(root_url)
-    end
+
```

Comment:
In general throughout all of our other specs, we have been including whitespace above it blocks. For this particular file, I did not add whitespace above the existing it blocks because I try not to make too many 'cleanup' changes outside the scope of the PR, as I find it can make the intention behind changes unclear. In this case, since it is one line and we have some context to the change, I agree that it is a good idea to include.